tests/run-make/compiler-builtins/rmake.rs

```diff
@@ -38,6 +38,14 @@ fn main() {
     let target_dir = tmp_dir().join("target");
     let target = std::env::var("TARGET").unwrap();
 
+    if target.starts_with("x86_64-fortanix-unknown-sgx") {
```

Comment:
I believe you can use a compiletest directive in rmake.rs:

//@ ignore-sgx

(please leave either a reason/fixme too)

Reply:
done

Reply:
(you shouldn't need the target check and return in main anymore)

Reply:
Yeah sorry. I forgot to remove it in the previous push. I have made the changes. Does it look fine now? @jieyouxu

Reply:
Yeah LGTM now